runtime/testy/studentAPI.py

```diff
@@ -155,3 +158,9 @@ def _print(self, *args):
     print(*args)
     console_string = " ".join([str(arg) for arg in args])
     self.toManager.put([SM_COMMANDS.SEND_CONSOLE, [console_string]]) 
+
+  def hibikeEnumerateDevices(self):
+    self.toManager.put([HIBIKE_COMMANDS.ENUMERATE, []])
+
+  def hibikeWriteValue(self, uid, params):
```

Comment:
we'll keep this in for now, and have #180 delete if necessary.
i believe you also stated that pull request is based on this, right?